src/Exception.h

```diff
@@ -99,8 +99,8 @@ class NullPointerException: public Exception
 	Exception( "Null pointer" )
 	{}
 
-    virtual ~NullPointerException() throw()
-	{}
+    virtual ~NullPointerException() throw() override
```

Comment:
Why? This `override´ does absolutely nothing; except being in the way and making somebody reading the code think what it might do. The answer is: Nothing.

You cannot override a destructor the wrong way: A destructor cannot have parameters, and there is no return value. And if you try to omit that `throw()`, the compiler will not complain, with or without `override`. I even tried it.

So let's not keep adding stuff that only distracts.